wremnants/theory_corrections.py

```diff
@@ -17,10 +17,12 @@
 def valid_theory_corrections():
     corr_files = glob.glob(common.data_dir+"TheoryCorrections/*Corr*.pkl.lz4")
     matches = [re.match("(^.*)Corr[W|Z]\.pkl\.lz4", os.path.basename(c)) for c in corr_files]
-    return [m[1] for m in matches if m]
+    return [m[1] for m in matches if m] + ['none']
```

Comment:
I think this can be removed, if one don't want to use a theory correction one can specify "--theoryCorr" without an argument